src/together/resources/finetune.py

```diff
@@ -493,6 +519,73 @@ def create(
 
         return FinetuneResponse(**response.data)
 
+    def estimate_price(
+        self,
+        *,
+        training_file: str,
+        model: str | None,
```

Comment:
Can these fields be none?

Reply:
They can in principle, but the price estimation API doesn't seem to support it. Therefore, we need to fix the API, disable calling the estimation when model is `None`, but set this field to always be defined (i.e., remove the None option)